risposte_gabriele.txt

```diff
@@ -0,0 +1,29 @@
+
+1) Un sistema di versioning permette di registrare i cambiamenti che vengono fatti all'interno di un file o di più file. Viene creata una history delle varie versioni e permette la navigazione all'interno di esse.
+Facilita il lavoro di più persone all'interno di un singolo progetto, permette di commentare, salvare ogni modifica eseguita.
+
+2) git init
```

Comment:
con git init crei un repo in locale ex novo; puoi scaricare in locale un repo già esistente anche con il comando git clone